tests/functional/forklift/test_legacy.py

```diff
@@ -13,6 +13,24 @@
 import pytest
 
 
+def test_incorrect_post_redirect(webtest):
+    """
+    Per issue #8104, we should issue an HTTP-308 for a POST
+    in /legacy and point the user to the correct endpoint,
+    /legacy/
+
+    See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/308
+    """
+    resp = webtest.post("/legacy", status=308)
```

Comment:
improvement: I've been trying to use more descriptive statuses from `http` since I can't ever recall the specific codes.

needs `from http import HTTPStatus`

```suggestion
    resp = webtest.post("/legacy", status=HTTPStatus.PERMANENT_REDIRECT)
```

Reply:
OK, I've updated this everywhere.